cvcreator/vitae/tech_skills.py

```diff
@@ -92,3 +88,49 @@ def make_skill_groups(
     output = [TechnicalSkill(title=title, values=output[title])
               for title in skills_data["allowed_labels"] if title in output]
     return output
+
+
+def get_skills(skill_categories: List[SkillCategory]) -> List[TechnicalSkill]:
+    """Process skills organized in skill categories into a list of `TechnicalSkill` objects
+
+    Args:
+        skill_categories:
+            User provided skills to distribute into groups.
+
+    Returns:
+        List of TechnicalSkill objects
+    """
+    output: List[TechnicalSkill] = []
+
+    # Get technical skills data
+    skills_data = get_skills_data()
+
+    # Process the skill categories in the CV
+    for skcat in skill_categories:
+        label = skcat.category
+        skills = skcat.technical_skills
+        # Validation of skill categories and skills
+        assert label in skills_data["allowed_labels"], ValueError(f"{label}: Invalid skill label")
```

Comment:
I suggest changing this line to raise the ValueError explicitly:

```suggestion
        if label not in skills_data["allowed_labels"]:
            raise ValueError(f"{label}: Invalid skill label")
```

Reply:
Totally. I was just very conservative with my changes to increase the chances of getting this PR done :)